doc/nrf/ug_thread_tools.rst

```diff
@@ -53,7 +53,16 @@ The interaction is possible using commands proper to wpanctl, a module installed
 Installing wpantund
 ===================
 
-For installation and initial configuration, see `wpantund Installation Guide`_.
+To ensure that the interaction with the samples works as expected, you should install the version of wpantund that has been used for testing the |NCS|.
```

Comment:
Proposing something like this: To ensure..., install the verified version of ``wpantund``.

Reply:
Change to "install", but I think we need to define what we mean with "verified" - since we haven't really verified the tool, but have only used it to test our samples.